src/tedpy/ted6000.py

```diff
@@ -26,13 +27,21 @@
 class TED6000(TED):
     """Instance of TED6000."""
 
+    class SystemType(Enum):
+        """Defines the various TED6000 System Types"""
+
+        NET = 0
+        NET_GEN = 1
+        NET_LOAD = 2
+
```

Comment:
Let's move this to `dataclasses` to be consistent with the other enums.

Reply:
Moved.